### ISSUE TITLE
Feature request: Keyboard navigation

### ISSUE THREAD

**Issue by dybber** [SpiderOak/Encryptr#272]
Currently you can login and search for passwords using the keyboard only, but you cannot:

 - select between various search results using keyboard
 - enter the Content Item Detail view using keyboard
 - select between various attributes in the Content Item Detail view with the keyboard
 - copy a password or other attribute using keyboard

**Comment by devgeeks** [comment]
Yeah, this is something we have always wanted.

There are some updates coming for the app, maybe some of these could happen then.
